transmart-rest-api/grails-app/controllers/org/transmartproject/rest/ExportController.groovy

```diff
@@ -205,7 +206,11 @@ class ExportController {
      */
     def fileFormats(@RequestParam('dataView') String dataView) {
         checkForUnsupportedParams(params, ['dataView'])
-        def fileFormats = restExportService.getSupportedFormats(dataView)
+        def view = DataView.from(dataView)
```

Comment:
if the data view is case-insensitive, maybe it should be also changed in [line 88](https://github.com/thehyve/transmart-core/blob/bd5471608a38dd2bf1ac743a81ead1eeddde0301/transmart-rest-api/grails-app/controllers/org/transmartproject/rest/ExportController.groovy#L88)

Reply:
Good catch. Would have been picked up with static compilation, but grails...